README.md

```diff
@@ -105,15 +110,15 @@ environments.  If you would like to use nix, you can run the following playbook.
 ansible-playbook \
     -i ansible/inventories/dev.yml \
     -e ip=$ip \
-    --key-file <path-to-your-pem-file> \
+    --key-file $key_file \
     ./ansible/playbooks/init-nix.yml
 ```
 
 Now, you can ssh into the instance and start hacking away on your project
 with nitro.
 
 ```bash
-ssh -i <path-to-your-pem-file> ec2-user@$ip
+ssh -i $key_file ec2-user@$instance_ip
```

Comment:
This is the main point of the PR, the update of `$ip` to `$instance_ip`, which should get merged, but the rest is optional, IMO a convenience for the reader

Reply:
this is great thank you for the updates.